integration-tests/smoke/keeper_test.go

```diff
@@ -202,7 +240,7 @@ func getKeeperSuite(registryVersion ethereum.KeeperRegistryVersion, registryConf
 						latestKeeper := upkeepInfo.LastKeeper
 						g.Expect(latestKeeper).ShouldNot(Equal(actions.ZeroAddress.String()), "Last keeper should be non zero")
 						g.Expect(latestKeeper).Should(BeElementOf(keepersPerformed), "Existing keepers should alternate turns wihtin BCPT")
-					}, "30s", "1s").Should(Succeed())
```

Comment:
Did you increase the running time for consistency reasons to have 1 minute everywhere?